python/createXSworkspace.py

```diff
@@ -846,4 +853,5 @@ def createXSworkspace(obsName, channel, nBins, obsBin, observableBins, usecfacto
 #createXSworkspace("pT4l", "2e2mu", 4, 1, False, True)
 #createXSworkspace("pT4l", "2e2mu", 4, 2, False, True)
 #createXSworkspace("pT4l", "2e2mu", 4, 3, False, True)
-#createXSworkspace("mass4l", "2e2mu", 2, 0, ["105.0","140.0"], False, True, "ggH_powheg_JHUgen_125", "v2","2018")
+#def createXSworkspace(obsName, channel, nBins, obsBin, observableBins, usecfactor, addfakeH, modelName, physicalModel, year, obs_ifJES, obs_ifJES2, obs_ifAbs, obs_ifAbs2, zzFloatType = '') 
```

Comment:
This one

Reply:
retained a test command, removed the line with `def`